### DISCUSSION TITLE
Guided Images: Images don't match - any docs?

### DISCUSSION THREAD

**Discussion by DrGunnarMallon** [deforum-art/deforum-stable-diffusion#281] [Q&A]
Hi All,

Is there some documentation on how to use guided images? I've been following the guide in the UI and keep getting "error: images don't match".

This is my guided images json:

{
    "0": "https://cdn.pixabay.com/photo/2020/05/29/23/09/peacock-butterfly-5236979_960_720.jpg",
    "99": "https://cdn.pixabay.com/photo/2020/05/29/23/09/peacock-butterfly-5236979_960_720.jpg"
}

Any help would be greatly appreciated on how to make my video loop back to the starting image. Thank you!

**Comment by DrGunnarMallon** [comment]
Doh, the guided image and the video need to have the same dimensions.

**Comment by dangerweenie** [comment]
hey curious if you've had any luck consistently hitting each image - i'm unable to get the animation to match anything but the first image.
